synthetics/data_source_agent.go

```diff
@@ -21,11 +20,11 @@ func dataSourceAgent() *schema.Resource {
 }
 
 func dataSourceAgentRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	tflog.Debug(ctx, fmt.Sprintf("Kentik API request - read agent with ID: %s\n", d.Get(idKey).(string)))
+	tflog.Debug(ctx, "Get synthetic Kentik API request", "ID", d.Get(idKey).(string))
```

Comment:
```suggestion
	tflog.Debug(ctx, "Get Agent Kentik API request", "ID", d.Get(idKey).(string))
```
The name of the resource is Agent.
In other places too.

Reply:
Done